api/src/main/java/jakarta/resource/ResourceException.java

```diff
@@ -135,6 +136,7 @@ public Exception getLinkedException() {
      * <code>initCause</code> methods of the 
      * <code>java.lang.Throwable</code> class.
      */
+    @Deprecated
```

Comment:
When deprecating, you should tell the user what you expect them to use instead.

Reply:
I do not deprecate anything, see `@javadoc` in javadoc. What do you want the person who just follows compiler advice to add there?